muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/notes/NotesRESTService.java

```diff
@@ -163,8 +210,10 @@ public Response archiveNote(@PathParam ("NOTEID") Long noteId) {
       }
     }
 
-      notesController.archiveNote(note);
-      
-      return Response.noContent().build();
+    Note updatedNote = notesController.toggleArchived(note);
+    if(note.getArchived().equals(Boolean.FALSE)) {
```

Comment:
Is this block useful for something? duedate=null surely isn't saved to database, it might go to the rest model though.